core/geometry/src/Geometry.ts

```diff
@@ -872,18 +988,26 @@ export class Geometry {
       return maxCount;
     return stepCount;
   }
-  /** Test if x is in simple 0..1 interval.  But optionally skip the test.  (this odd behavior is very convenient for code that sometimes does not do the filtering.)
+  /**
+   * Test if x is in simple 0..1 interval. But optionally skip the test. (this odd behavior is very
+   * convenient for code that sometimes does not do the filtering.)
    * @param x value to test.
    * @param apply01 if false, accept all x.
    */
-  public static isIn01(x: number, apply01: boolean = true): boolean { return apply01 ? x >= 0.0 && x <= 1.0 : true; }
-  /** Test if x is in simple 0..1 interval.  But optionally skip the test.  (this odd behavior is very convenient for code that sometimes does not do the filtering.)
+  public static isIn01(x: number, apply01: boolean = true): boolean {
+    return apply01 ? x >= 0.0 && x <= 1.0 : true;
+  }
+  /**
+   * Test if x is in simple 0..1 interval. But optionally skip the test. (this odd behavior is very convenient
+   * for code that sometimes does not do the filtering.)
    * @param x value to test.
    * @param apply01 if false, accept all x.
    */
```

Comment:
This function doesn't exhibit the optional odd behavior (unlike `isIn01`), so the doc can be simplified and `@param apply01` can be removed: "Test if `x` is in the interval [0,1] with tolerance" or some such.